data/presets/presets/indoor/elevator.json

```diff
@@ -0,0 +1,20 @@
+{
+    "icon":"temaki-elevator",
+    "fields": [
+        "level",
```

Comment:
You can use the `level_semi` field I added recently. This lets mappers more easily specify that a feature spans multiple floors.